src/style-spec/function/definitions/curve.js

```diff
@@ -17,6 +16,11 @@ export type InterpolationType =
     { name: 'exponential', base: number } |
     { name: 'cubic-bezier', controlPoints: [number, number, number, number] };
 
+export type InterpolationCache = {
+    pow: {[number]: {[number]: number}},
+    unitBezier: {[string]: UnitBezier}
+}
```

Comment:
I'm wondering if this kind of cache gives us any perf wins given the increased complexity... If it's marginal, maybe we should take this out and revisit later? E.g. UnitBezier [doesn't do much in the constructor](https://github.com/mapbox/unitbezier/blob/master/index.js#L33-L39) compared to the main approximation code. And `Math.pow` values are cached by non-integer `exp` values which are known to be much slower in V8 than integer keys.

Reply:
@mourner ah, yeah: just realzied that the UnitBezier caching is a holdover from when I was using https://github.com/gre/bezier-easing, which does some precomputation upon construction.

Re: `Math.pow`, I did, in fact see a small but definitely visible gain from adding this caching -- something 2% on the tile_layout_dds benchmark.